.storybook/webpack.config.js

```diff
@@ -123,7 +108,7 @@ module.exports = (storybookBaseConfig, configType) => {
               }),
               postcssCustomProperties({
                 preserve: false,
-                importFrom: materialSources,
+                importFrom: 'materials/exports/vars.css',
```

Comment:
I don't like these names but I'm not feeling very creative 😞

Reply:
I would rename `imports.js/css` to `custom-properties.js/css`. That name seems more appropriate. "custom properties" is more or less a fancy name for "css variables". And that's what we're exposing here.